docs/guides/observability/grafana-dashboards.adoc

```diff
@@ -0,0 +1,91 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="{project_name} Grafana dashboards"
+summary="Learn how to import {project_name} provided Grafana dashboards to your Grafana instance" >
+
+{project_name} provides metrics to observe what is happening inside the deployment.
+To understand how metrics evolve over time, it is helpful to collect them and visualize in graphs.
+
+
+This guide provides instructions on how to visualize collected {project_name} metrics in a running Grafana instance.
+
+== Prerequisites
+
+* {project_name} metrics are enabled. Follow <@links.observability id="configuration-metrics"/> {section} for more details.
+* Grafana instance is running and {project_name} metrics are collected into a Prometheus instance.
+
+== Keycloak Grafana dashboards
+
+Grafana dashboards are distributed in form of a JSON file that is imported into a Grafana instance.
+JSON definitions of {project_name} Grafana dashboards are available in the following https://github.com/keycloak/keycloak-grafana-dashboard[Github repository].
+
+Follow these steps to download JSON file definitions.
+
+. Identify the branch from `keycloak-grafana-dashboards` to use from the following table.
++
+
+|===
```

Comment:
Maybe this should be described in the README.md of the repository as this docs might be eventually for an older release, and then `main` would have continued to evolve for a different release.

Reply:
I was thinking about this as well. My reasoning to put it here is that I assume it won't change very often and would be better to have everything in one place rather than documenting it in the repo. We most probably won't make any breaking changes at least until KC27. Also, I think the way how the mapping works may change in the future, for example, we will have different mapping per dashboard. Do you still think it makes sense to move it?

Reply:
Thanks, I think it's okay to keep it here for now.